clang/docs/ReleaseNotes.rst

```diff
@@ -390,6 +390,10 @@ Bug Fixes to C++ Support
   we now produce a diagnostic. Fixes:
   (`#65522 <https://github.com/llvm/llvm-project/issues/65522>`_)
 
+- Fixed a bug where clang couldn't choose the correct deduction guide from
+  two implicitly generated deduction guides. One of them generated from a
+  non-templated constructor and the other from a templated constructor.
+
```

Comment:
Perhaps reword to:

"Fixed a bug where clang incorrectly considered implicitly generated deduction guides from a non-templated constructor and a templated constructor as ambiguous, rather than prefer the non-templated constructor as specified in [standard.group]p3."